3rd_party/3rd_party.cmake

```
@@ -242,14 +242,15 @@ if (GCC_RT_LOCATION)
   foreach(LIBRARY ${FOUND_LIBRARIES})
     execute_process(COMMAND patchelf --print-rpath ${LIBRARY} COMMAND grep lib OUTPUT_VARIABLE RPATH_VAR ERROR_VARIABLE RPATH_ERR WORKING_DIRECTORY "${INSTALL_DIR}" OUTPUT_STRIP_TRAILING_WHITESPACE)
     if(RPATH_VAR)
-      execute_process(COMMAND patchelf --force-rpath --set-rpath "$ORIGIN" ${RES} OUTPUT_VARIABLE SET_RPATH_OUT ERROR_VARIABLE SET_RPATH_ERR WORKING_DIRECTORY "${INSTALL_DIR}" OUTPUT_STRIP_TRAILING_WHITESPACE)
+        message(STATUS "Attempting to overwrite existing RPATH ${RPATH_VAR} in ${LIBRARY}")
```

nit: the indenting is slightly off here

```suggestion
      message(STATUS "Attempting to overwrite existing RPATH ${RPATH_VAR} in ${LIBRARY}")
```